mysql-test/suite/sys_vars/t/innodb_encryption_threads_basic.test

```diff
@@ -1,5 +1,45 @@
---source include/have_innodb.inc
+--disable_warnings
+SET GLOBAL innodb_file_per_table = ON;
+INSTALL PLUGIN keyring_file SONAME 'keyring_file.so';
+--enable_warnings
 
---skip Not implemented
+SET GLOBAL innodb_encryption_threads = 10;
+SELECT @@global.innodb_encryption_threads;
+
+SET GLOBAL innodb_encryption_threads = 1000;
+SELECT @@global.innodb_encryption_threads;
+
+SET GLOBAL innodb_encryption_threads = -1000;
+SELECT @@global.innodb_encryption_threads;
+
+--error ER_WRONG_TYPE_FOR_VAR
+SET GLOBAL innodb_encryption_threads = 10.5;
+SELECT @@global.innodb_encryption_threads;
+
+--error ER_WRONG_TYPE_FOR_VAR
+SET GLOBAL innodb_encryption_threads = ON;
+SELECT @@global.innodb_encryption_threads;
+
+--error ER_WRONG_TYPE_FOR_VAR
+SET GLOBAL innodb_encryption_threads = 'a';
+SELECT @@global.innodb_encryption_threads;
```

Comment:
I think you forgot the string too . "abc" ?

Reply:
Will not "innodb_encryption_threads = ON" count as string?

Reply:
ON/OFF are mapped to bools. Please use explicit string

Reply:
Done